docs/faq.md

```diff
@@ -16,3 +16,14 @@ We list some potential troubles encountered by users and developers, along with
   pip uninstall pycocotools mmpycocotools
   pip install mmpycocotools
   ```
+
+- If you face the error shown below, and your environment contains numba == 0.48.0 with numpy >= 1.20.0
+
+  ``TypeError: expected dtype object, got 'numpy.dtype[bool_]'``
+
+  please downgrade numpy to 1.18.0 or install numba == 0.48 from source, because numpy 1.20.0 'np.dtype' now produces subclass. This is an API change. The current workaround is as below.
```

Comment:
The numpy version must be 1.18.0? Could it be 1.19.0?

Reply:
it can be 1.19.0.  so should I change 1.18.0 to 1.19.0?  The reason I written 1.18.0 was that it could match this [issue](https://github.com/numba/numba/issues/6041) in numba

Reply:
Then we should not force the numpy version, rather, we should give a valid range for that. Further, if numpy >=1.20.0, is there any numba version greater than 0.48.0 can fix the issue?

Reply:
if we want to use the numba version greater than 0.48.0, we need to modify some code. So we can only choose one workaround.